models/Discord/Commands/DiscordCommandInfo.js

```diff
@@ -31,7 +39,7 @@ class DiscordCommandInfo extends DiscordCommand {
 				embed.addField("Players online:", results.data, true);
 				embed.addField("Current round:", byondSS.roundNumber, true);
 				embed.addField("Admins online:", adminwho, false);
-				embed.setColor("9541a5");
+				embed.setColor(colour);
```

Comment:
embed.set**Color**(**colour**);

Pliz, Im aware that americans are simple, but consistency.... eeeee